blog/models.py

```diff
@@ -0,0 +1,69 @@
+from django.db import models
+from django.contrib.auth.models import User
+from tinymce import models as tinymce_models
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255)
+    body = tinymce_models.HTMLField()  # wysiwyg
+    author = models.ForeignKey(User, on_delete=models.PROTECT)
+    created = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+
```

Comment:
any of these other classes can be changed to look more like the code commented in above